models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -98,44 +98,7 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
   
   param <- read.table(template.param)
   
-  #### write INITAL CONDITIONS here ####
-  if (!is.null(IC)) {
-    ic.names <- names(IC)
-    ## plantWoodInit gC/m2
-    if ("plantWood" %in% ic.names) {
-      param[which(param[, 1] == "plantWoodInit"), 2] <- IC$plantWood
-    }
-    ## laiInit m2/m2
-    if ("lai" %in% ic.names) {
-      param[which(param[, 1] == "laiInit"), 2] <- IC$lai
-    }
-    ## litterInit gC/m2
-    if ("litter" %in% ic.names) {
-      param[which(param[, 1] == "litterInit"), 2] <- IC$litter
-    }
-    ## soilInit gC/m2
-    if ("soil" %in% ic.names) {
-      param[which(param[, 1] == "soilInit"), 2] <- IC$soil
-    }
-    ## litterWFracInit fraction
-    if ("litterWFrac" %in% ic.names) {
-      param[which(param[, 1] == "litterWFracInit"), 2] <- IC$litterWFrac
-    }
-    ## soilWFracInit fraction
-    if ("soilWFrac" %in% ic.names) {
-      param[which(param[, 1] == "soilWFracInit"), 2] <- IC$soilWFrac
-    }
-    ## snowInit cm water equivalent
-    if ("snow" %in% ic.names) {
-      param[which(param[, 1] == "snowInit"), 2] <- IC$snow
-    }
-    ## microbeInit mgC/g soil
-    if ("microbe" %in% ic.names) {
-      param[which(param[, 1] == "microbeInit"), 2] <- IC$microbe
-    }
-  }else{
-    #some stuff about IC file that we can give in lieu of actual ICs
```

Comment:
My intent was for you to insert the IC.file stuff here, as a second alternative, not for you to delete the current IC arg, which we need to retain for SDA (at least for the time being)